ddtrace/contrib/django/patch.py

```diff
@@ -411,7 +411,23 @@ def traced_get_response(django, pin, func, instance, args, kwargs):
                     span.error = 1
                 span.set_tag("django.response.class", func_name(response))
                 if hasattr(response, "template_name"):
-                    utils.set_tag_array(span, "django.response.template", response.template_name)
+                    # template_name is a bit of a misnomer, as it could be any of:
+                    # a list of strings, a tuple of strings, a single string, or an instance of Template
+                    # for more detail, see:
+                    # https://docs.djangoproject.com/en/3.0/ref/template-response/#django.template.response.SimpleTemplateResponse.template_name
+                    template = response.template_name
+
+                    if isinstance(template, six.string_types):
+                        template_names = [template]
+                    elif isinstance(template, (list, tuple,)):
+                        template_names = template
```

Comment:
would an `isinstance(template, Template)` be better here?

Reply:
that was my first thought as well, but unfortunately the different template backend classes don't have a common base and there are some variances across the different django versions, so I opted for checking the attribute instead